engines/wintermute/base/gfx/opengl/base_render_opengl3d.h

```diff
@@ -152,6 +166,9 @@ class BaseRenderOpenGL3D : public BaseRenderer {
 					  float angle, uint32 color, bool alphaDisable, Graphics::TSpriteBlendMode blendMode, bool mirrorX, bool mirrorY);
 
 private:
+	Math::Matrix4 _lastProjectioMatrix;
+	Math::Matrix4 _lastViewMatrix;
+	float _fov;
```

Comment:
I'd feel safer if this was zero-initialized in the constructor, even if it's initialized in setup3D. Static analysis tools will probably not be too happy about it.